src-packed/validator.js

```diff
@@ -62,12 +63,31 @@ export async function getMatches(text, matches) {
         console.log(result.error);
         return;
     }
-    result.sentences.forEach(sentence => {
+
+    for (var i = 0; i < result.sentences.length; i++) {
+        var sentence = result.sentences[i];
         if (sentence.sentiment !== "negative")
-            return;
+            continue;
         console.log(`Index: ${sentence.offset}, Negative sentiment: ${sentence.confidenceScores.negative}`)
         if (sentence.confidenceScores.negative <= 0.75)
-            return;
+            continue;
+
+        // Remove commonly used phrases with a negative sentiment
+        var regex = new RegExp(ignoreList.ignorablePhrases.join("|"), "gi");
+        var trimmedText = sentence.text.replace(regex, "");
+        // Analyze sentiment again if a commonly used phrase with a negative sentiment was removed
+        if (trimmedText !== sentence.text) {
+            const trimmedResult = await textAnalytics.analyzeSentiment(trimmedText);
```

Comment:
This seems like it would hit the API again and make a web request?

Is there a way, we can just return early instead, and not add to the `matches` list?